Gems/Atom/Feature/Common/Assets/Materials/Types/NewPBR/NewPBR.azsli

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
```

Comment:
What does 'NewBPR' stand for? It is not clear what makes it new and I suggest to either rename or to have a one sentence comment that shed light on the naming / the difference.